docs/upgrade-to-v4.md

```diff
@@ -0,0 +1,67 @@
+# Upgrade to v4
+
+The latest release of `bson` represents the first major refactoring of the module, and introduces a
+number of breaking changes. The following guide lists these changes, and hopes to aid migration and
+adoption.
+
+## Changelog
+
+You can view the [Changelog](https://github.com/mongodb/js-bson/blob/master/HISTORY.md) for a
+detailed list of the changes in this release.
+
+## Breaking Changes
+
+### Node
+
+To use new ES2015 features, bson v4 requires at least Node v6 or above.
+
+### BSON
+
+* You no longer need to (or can) create an instance of the `BSON` object. Instead, `BSON` acts as a
```

Comment:
This phrasing is a little awkward. It reads more easily as "You can no longer create an instance of the BSON object."